packages/client/lib/util/parse.ts

```diff
@@ -166,15 +166,12 @@ async function createGethGenesisBlockHeader(json: any) {
  * @returns genesis parameters in a `CommonOpts` compliant object
  */
 async function parseGethParams(json: any) {
-  const { name, config, difficulty, nonce, mixHash, coinbase } = json
+  const { name, config, difficulty, nonce, mixHash, coinbase, baseFeePerGas } = json
 
-  let { gasLimit, extraData, baseFeePerGas, timestamp } = json
+  let { gasLimit, extraData, timestamp } = json
 
   // geth stores gasLimit as a hex string while our gasLimit is a `number`
   json['gasLimit'] = gasLimit = parseInt(gasLimit)
-  // geth assumes an initial base fee value on londonBlock=0
-  json['baseFeePerGas'] = baseFeePerGas =
-    baseFeePerGas === undefined && config.londonBlock === 0 ? 1000000000 : undefined
```

Comment:
Hmm upon thought, do we want to keep this default value in case it is not present in the json? @ryanio

Reply:
If I understand this correctly, Ryan already handled it by added that logic here https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/block/src/header.ts#L267-L272, being the default value defined here https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/common/src/eips/1559.json#L19 (which is the same as the previous code we had here) - Probably I am wrong tho :thinking:

Reply:
Nope, good point, you are right 👍 😄